--- original
+++ modified
@@ -1803,12 +1803,26 @@
 gosub inl[:LABEL_76]
 mov 0x14, stor[23]
 wait stor[23]
+jmp_cmp ==, var[34][2], 0xffffffff, inl[:SKIP_P_2]
 gosub inl[:LABEL_134]
+SKIP_P_2:
 mov 0x5, stor[23]
 wait stor[23]
+jmp_cmp ==, var[34][1], 0xffffffff, inl[:SKIP_P_1]
 gosub inl[:LABEL_131]
+SKIP_P_1:
 mov 0x7, stor[23]
 wait stor[23]
 gosub inl[:LABEL_137]
 mov 0x8, stor[23]
 wait stor[23]
+mov 0x3, stor[4]
+LOOP_0:
+mov var[34][stor[4]], stor[5]
+jmp_cmp ==, stor[5], 0xffffffff, inl[:SKIP]
+gosub inl[:FORK_TO_FALL]
+mov 0x1, stor[23]
+wait stor[23]
+incr stor[4]
+jmp inl[:LOOP_0]
+SKIP: 
@@ -2496,15 +2510,49 @@
 LABEL_141:
 call 548, stor[0], 0xffffffff, stor[24], stor[25], stor[26]
 mov stor[24], stor[27]
-mov var[34][0], stor[28]
+mov 0x0, stor[4]
+LOOP:
+mov var[34][stor[4]], stor[28]
+jmp_cmp ==, stor[28], 0xffffffff, inl[:SKIP_R] 
 gosub inl[:LABEL_142]
+incr stor[4]
+jmp inl[:LOOP]
+SKIP_R:
-mov var[34][1], stor[28]
-gosub inl[:LABEL_142]
-mov var[34][2], stor[28]
-gosub inl[:LABEL_142]
 yield
 jmp inl[:LABEL_141]
 data 0x1
+FORK_TO_FALL:
+call 600, stor[18]
+mov stor[4], stor[stor[stor[0], 18], 4]
+call 618, stor[18], inl[:FALL_FORK]
+return
+FALL_FORK:
+mov var[34][stor[4]], stor[28]
+call 516, stor[28]
+call 606, stor[18], 0x18a05
+call 595, stor[18], 0x1
+call 549, stor[18], 0xffffffff, 0x200, 0x200, 0x200
+call 545, stor[18], 0xffffffff, 0x0, 0x0, 0x0
+call 545, stor[18], stor[28], 0x0, 0xfffffc00, 0xffffe000
+call 558, stor[18], 0xffffffff, 0x30d40, 0x0, 0x0, 0x0, 0x0, 0x0
+mov 0x1, stor[23]
+wait stor[23]
+gosub inl[:LABEL_16]
+call 545, stor[18], stor[28], 0x0, 0xfffffc00, 0x0
+call 549, stor[18], 0xffffffff, 0x2000, 0x2000, 0x2000
+call 236, 0xa, 0x6, 0x0, 0x0, 0x0, 0x0
+call 322, stor[28], 0x1b
+mov stor[stor[stor[0], 28], 24], stor[24]
+call 146, stor[28], 0x0
+add 0x1000, stor[24]
+mov 0x1, stor[23]
+FALL_LOOP:
+call 148, stor[28], 0xffffffff, 0xa, stor[24], stor[stor[stor[0], 28], 25], stor[stor[stor[0], 28], 26]
+yield
+while stor[23], inl[:FALL_LOOP]
+call 640, 0x1, 0xfffffe00, 0x400, 0x0, stor[24], stor[25], stor[26]
+call 130, stor[28], stor[24], stor[25], stor[26]
+rewind
 LABEL_142:
 call 513, stor[28], stor[27]
 mov stor[stor[stor[0], 28], 24], stor[24]