--- original
+++ modified
@@ -1473,23 +1473,19 @@
 call 588, stor[18], 0x19
 call 238, 0x2, stor[17], 0x1, 0x0, 0x0, 0x0, 0x0
-gosub inl[:LABEL_26]
-call 600, stor[18]
-mov stor[27], stor[stor[stor[0], 18], 27]
-mov stor[28], stor[stor[stor[0], 18], 28]
-call 618, stor[18], inl[:LABEL_31]
-yield
 call 600, stor[18]
 mov stor[29], stor[stor[stor[0], 18], 29]
 mov 0x2, stor[stor[stor[0], 18], 23]
 call 618, stor[18], inl[:LABEL_33]
-gosub inl[:LABEL_24]
+mov 0x0, stor[30]
+mov var[34][stor[30]], stor[28]
+LOOP:
 call 600, stor[18]
 mov stor[27], stor[stor[stor[0], 18], 27]
 mov stor[28], stor[stor[stor[0], 18], 28]
 call 618, stor[18], inl[:LABEL_31]
 yield
-gosub inl[:LABEL_25]
-call 600, stor[18]
-mov stor[27], stor[stor[stor[0], 18], 27]
-mov stor[28], stor[stor[stor[0], 18], 28]
-call 618, stor[18], inl[:LABEL_31]
+incr stor[30]
+mov var[34][stor[30]], stor[28]
+jmp_cmp !=, stor[28], 0xffffffff, inl[:LOOP]
 mov 0x2e, stor[23]
 wait stor[23]