--- original
+++ modified
@@ -862,7 +862,7 @@
 div 0xa, stor[9]
 jmp_cmp >=, stor[9], stor[8], inl[:LABEL_6]
 call 371, stor[28], stor[10], 0x0
-add 0x81, stor[10]
+add var[45][136], stor[10]
 mov var[45][stor[10]], stor[9]
 jmp_cmp !&, 0x80, stor[9], inl[:LABEL_3]
 and 0x7, stor[9]
@@ -3282,7 +3282,7 @@
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_38]
 mov 0x0, stor[11]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov var[45][stor[9]], stor[10]
 jmp_cmp !&, 0x80, stor[10], inl[:LABEL_39]
 and 0xf, stor[10]
@@ -3305,7 +3305,7 @@
 call 600, stor[18]
 mov stor[0], stor[stor[stor[0], 18], 28]
 mov stor[32], stor[stor[stor[0], 18], 31]
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov stor[32], var[45][stor[9]]
 or 0x480, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
@@ -3747,7 +3747,7 @@
 data 0x49
 LABEL_58:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov 0x0, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
 and 0xfffffff0, stor[8]
