--- original
+++ modified
@@ -24,7 +24,7 @@
 mov 0x20, var[128 + stor[8]][5]
 mov 0x0, var[128 + stor[8]][6]
 mov 0x0, var[128 + stor[8]][7]
-add 0x81, stor[8]
+add var[45][136], stor[8]
 mov 0x0, var[45][stor[8]]
 mov var[45][0], stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_2]
@@ -262,7 +262,7 @@
 LABEL_14:
 mov var[32][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_15]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -275,7 +275,7 @@
 LABEL_17:
 mov var[32][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 gosub inl[:LABEL_119]
 incr stor[11]
 jmp_cmp <, stor[11], var[33], inl[:LABEL_17]
@@ -621,3 +621,3 @@
 call 371, stor[28], stor[8], 0x1
-call 128, stor[28], inl[:LABEL_65[stor[8]]], 0x0, inl[:LABEL_66[stor[8]]]
+call 128, stor[28], stor[stor[stor[0], 28], 24], stor[stor[stor[0], 28], 25], stor[stor[stor[0], 28], 26]
 call 130, stor[28], 0x0, 0x400, 0x0
@@ -1186,7 +1187,7 @@
 shr 0x1, stor[9]
 LABEL_91:
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 mov var[45][stor[10]], stor[10]
 jmp_cmp !&, 0x2, stor[10], inl[:LABEL_92]
 jmp_cmp &, 0x4, stor[10], inl[:LABEL_92]
@@ -1194,7 +1195,7 @@
 LABEL_92:
 jmp_cmp >=, 0x0, stor[9], inl[:LABEL_94]
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 jmp_cmp !&, 0x1, var[45][stor[10]], inl[:LABEL_94]
 call 177, stor[30], CURRENT_HP, stor[10]
 call 177, stor[30], MAX_HP, stor[11]
@@ -1744,7 +1745,7 @@
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp !&, 0x10, stor[8], inl[:LABEL_99]
 call 371, stor[0], stor[8], 0x0
-add 0x81, stor[8]
+add var[45][136], stor[8]
 or 0xf00, var[45][stor[8]]
 LABEL_99:
 return
@@ -1798,7 +1799,7 @@
 call 169, stor[0], GUARD, stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_104]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 call 169, stor[0], STATUS, stor[8]
 and 0xff, stor[8]
 jmp_cmp ==, 0x0, stor[8], inl[:LABEL_107]
@@ -1899,7 +1900,7 @@
 gosub inl[:LABEL_110]
 jmp_cmp >, 0x0, stor[8], inl[:LABEL_118]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp &, 0x80, var[45][stor[9]], inl[:LABEL_116]
 or 0x80, var[45][stor[9]]
 call 600, stor[18]
@@ -2300,7 +2301,7 @@
 return
 LABEL_145:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_146]
 or 0x40, var[45][stor[9]]
 LABEL_146:
