--- original
+++ modified
@@ -24,7 +24,7 @@
 mov 0x20, var[128 + stor[8]][5]
 mov 0x0, var[128 + stor[8]][6]
 mov 0x0, var[128 + stor[8]][7]
-add 0x81, stor[8]
+add var[45][136], stor[8]
 mov 0x0, var[45][stor[8]]
 mov var[45][0], stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_2]
@@ -271,7 +271,7 @@
 LABEL_12:
 mov var[32][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_13]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -284,7 +284,7 @@
 LABEL_15:
 mov var[32][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 gosub inl[:LABEL_94]
 incr stor[11]
 jmp_cmp <, stor[11], var[33], inl[:LABEL_15]
@@ -1006,7 +1006,7 @@
 shr 0x1, stor[9]
 LABEL_66:
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 mov var[45][stor[10]], stor[10]
 jmp_cmp !&, 0x2, stor[10], inl[:LABEL_67]
 jmp_cmp &, 0x4, stor[10], inl[:LABEL_67]
@@ -1014,7 +1014,7 @@
 LABEL_67:
 jmp_cmp >=, 0x0, stor[9], inl[:LABEL_69]
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 jmp_cmp !&, 0x1, var[45][stor[10]], inl[:LABEL_69]
 call 177, stor[30], CURRENT_HP, stor[10]
 call 177, stor[30], MAX_HP, stor[11]
@@ -1564,7 +1564,7 @@
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp !&, 0x10, stor[8], inl[:LABEL_74]
 call 371, stor[0], stor[8], 0x0
-add 0x81, stor[8]
+add var[45][136], stor[8]
 or 0xf00, var[45][stor[8]]
 LABEL_74:
 return
@@ -1618,7 +1618,7 @@
 call 169, stor[0], GUARD, stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_79]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 call 169, stor[0], STATUS, stor[8]
 and 0xff, stor[8]
 jmp_cmp ==, 0x0, stor[8], inl[:LABEL_82]
@@ -1719,7 +1719,7 @@
 gosub inl[:LABEL_85]
 jmp_cmp >, 0x0, stor[8], inl[:LABEL_93]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp &, 0x80, var[45][stor[9]], inl[:LABEL_91]
 or 0x80, var[45][stor[9]]
 call 600, stor[18]
@@ -2120,14 +2120,14 @@
 return
 LABEL_120:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_121]
 or 0x40, var[45][stor[9]]
 LABEL_121:
 return
 LABEL_122:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov 0x0, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
 and 0xffffff00, stor[8]
