--- original
+++ modified
@@ -27,7 +27,7 @@
 mov 0x20, var[128 + stor[8]][5]
 mov 0x0, var[128 + stor[8]][6]
 mov 0x0, var[128 + stor[8]][7]
-add 0x81, stor[8]
+add var[45][136], stor[8]
 mov 0x0, var[45][stor[8]]
 mov var[45][0], stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_2]
@@ -265,7 +265,7 @@
 LABEL_15:
 mov var[32][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_16]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -278,7 +278,7 @@
 LABEL_18:
 mov var[32][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 gosub inl[:LABEL_145]
 incr stor[11]
 jmp_cmp <, stor[11], var[33], inl[:LABEL_18]
@@ -493,7 +493,7 @@
 mov 0x0, stor[31]
 mov 0x0, stor[16]
 call 371, stor[32], stor[8], 0x0
-add 0xbd, stor[8]
+add var[45][196], stor[8]
 jmp_cmp &, 0x2, var[45][stor[8]], inl[:LABEL_40]
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp &, 0x10, stor[8], inl[:LABEL_40]
@@ -1492,7 +1492,7 @@
 shr 0x1, stor[9]
 LABEL_112:
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 mov var[45][stor[10]], stor[10]
 jmp_cmp !&, 0x2, stor[10], inl[:LABEL_113]
 jmp_cmp &, 0x4, stor[10], inl[:LABEL_113]
@@ -1500,7 +1500,7 @@
 LABEL_113:
 jmp_cmp >=, 0x0, stor[9], inl[:LABEL_115]
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 jmp_cmp !&, 0x1, var[45][stor[10]], inl[:LABEL_115]
 call 177, stor[30], CURRENT_HP, stor[10]
 call 177, stor[30], MAX_HP, stor[11]
@@ -2036,7 +2036,7 @@
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp !&, 0x10, stor[8], inl[:LABEL_123]
 call 371, stor[0], stor[8], 0x0
-add 0x81, stor[8]
+add var[45][136], stor[8]
 or 0xf00, var[45][stor[8]]
 LABEL_123:
 return
@@ -2084,7 +2084,7 @@
 call 169, stor[0], GUARD, stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_130]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 call 169, stor[0], STATUS, stor[8]
 and 0xff, stor[8]
 jmp_cmp ==, 0x0, stor[8], inl[:LABEL_133]
@@ -2185,7 +2185,7 @@
 gosub inl[:LABEL_136]
 jmp_cmp >, 0x0, stor[8], inl[:LABEL_144]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp &, 0x80, var[45][stor[9]], inl[:LABEL_142]
 or 0x80, var[45][stor[9]]
 call 600, stor[18]
@@ -2586,7 +2586,7 @@
 return
 LABEL_171:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_172]
 or 0x40, var[45][stor[9]]
 LABEL_172:
@@ -2640,7 +2640,7 @@
 LABEL_174:
 mov var[32][stor[8]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_175]
 and 0xffffff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -2798,7 +2798,7 @@
 call 371, stor[0], stor[8], 0x0
 mov var[128 + stor[8]][3], stor[28]
 call 167, stor[28], stor[10]
-jmp_cmp !=, 0x0, stor[10], inl[:LABEL_194]
+jmp_cmp !=, stor[28], var[34][0], inl[:LABEL_194]
 or 0x2, var[45][31]
 or 0x4, var[45][31]
 or 0x20, stor[7]
@@ -2821,7 +2821,7 @@
 call 371, stor[0], stor[8], 0x0
 mov var[128 + stor[8]][3], stor[28]
 call 167, stor[28], stor[10]
-jmp_cmp !=, 0x0, stor[10], inl[:LABEL_195]
+jmp_cmp !=, stor[28], var[34][0], inl[:LABEL_195]
 or 0x2, var[45][31]
 or 0x4, var[45][31]
 or 0x20, stor[7]
 