--- original
+++ modified
@@ -191,7 +191,7 @@
 shr 0x1, stor[9]
 LABEL_22:
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 mov var[45][stor[10]], stor[10]
 jmp_cmp !&, 0x2, stor[10], inl[:LABEL_23]
 jmp_cmp &, 0x4, stor[10], inl[:LABEL_23]
@@ -199,7 +199,7 @@
 LABEL_23:
 jmp_cmp >=, 0x0, stor[9], inl[:LABEL_25]
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 jmp_cmp !&, 0x1, var[45][stor[10]], inl[:LABEL_25]
 call 177, stor[30], CURRENT_HP, stor[10]
 call 177, stor[30], MAX_HP, stor[11]
@@ -358,7 +358,7 @@
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp !&, 0x10, stor[8], inl[:LABEL_30]
 call 371, stor[0], stor[8], 0x0
-add 0x81, stor[8]
+add var[45][136], stor[8]
 or 0xf00, var[45][stor[8]]
 LABEL_30:
 return
@@ -1401,7 +1401,7 @@
 LABEL_53:
 mov var[32][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_54]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -1414,7 +1414,7 @@
 LABEL_56:
 mov var[32][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 gosub inl[:LABEL_213]
 incr stor[11]
 jmp_cmp <, stor[11], var[33], inl[:LABEL_56]
@@ -1649,14 +1649,12 @@
 mov var[128 + stor[9]][7], stor[10]
 shr 0x18, stor[10]
 jmp_cmp !=, 0x0, stor[10], inl[:JMP_7d70_0]
+jmp_cmp !=, 0x0, reg[10], inl[:JMP_7d70_0]
 gosub inl[:LABEL_82]
 rand 0x64, stor[8]
 jmp_cmp <, 0x64, stor[8], inl[:LABEL_76]
-call 497, 0xffffffff, stor[8]
-jmp_cmp ==, 0xffffffff, stor[8], inl[:LABEL_76]
-shl 0x18, stor[8]
-call 371, stor[0], stor[9], 0x0
-or stor[8], var[128 + stor[9]][7]
+call 497, null, reg[10]
+jmp_cmp ==, 0, reg[10], inl[:LABEL_76]
 mov 0x0, stor[8]
 mov 0x1e, stor[9]
 mov 0x0, stor[10]
@@ -2329,7 +2327,7 @@
 mov 0x0, stor[31]
 mov 0x0, stor[16]
 call 371, stor[32], stor[8], 0x0
-add 0xbd, stor[8]
+add var[45][196], stor[8]
 jmp_cmp &, 0x2, var[45][stor[8]], inl[:LABEL_134]
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp &, 0x10, stor[8], inl[:LABEL_134]
@@ -2943,7 +2941,7 @@
 call 169, stor[0], GUARD, stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_198]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 call 169, stor[0], STATUS, stor[8]
 and 0xff, stor[8]
 jmp_cmp ==, 0x0, stor[8], inl[:LABEL_201]
@@ -3044,7 +3042,7 @@
 gosub inl[:LABEL_204]
 jmp_cmp >, 0x0, stor[8], inl[:LABEL_212]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp &, 0x80, var[45][stor[9]], inl[:LABEL_210]
 or 0x80, var[45][stor[9]]
 call 600, stor[18]
@@ -3445,14 +3443,14 @@
 return
 LABEL_239:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_240]
 or 0x40, var[45][stor[9]]
 LABEL_240:
 return
 LABEL_241:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov 0x0, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
 and 0xffffff00, stor[8]
@@ -3607,9 +3605,9 @@
 mov var[128 + stor[25]][7], stor[26]
 shr 0x18, stor[26]
 and 0xff, stor[26]
-jmp_cmp ==, 0x0, stor[26], inl[:LABEL_251]
 jmp_cmp ==, 0x1, stor[26], inl[:LABEL_249]
-call 498, stor[26], stor[8]
+jmp_cmp ==, 0x0, reg[10], inl[:LABEL_251]
+call 498, reg[10], reg[10]
 jmp inl[:LABEL_250]
 LABEL_249:
 mov var[5], stor[8]
