--- original
+++ modified
@@ -191,7 +191,7 @@
 shr 0x1, stor[9]
 LABEL_22:
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 mov var[45][stor[10]], stor[10]
 jmp_cmp !&, 0x2, stor[10], inl[:LABEL_23]
 jmp_cmp &, 0x4, stor[10], inl[:LABEL_23]
@@ -199,7 +199,7 @@
 LABEL_23:
 jmp_cmp >=, 0x0, stor[9], inl[:LABEL_25]
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 jmp_cmp !&, 0x1, var[45][stor[10]], inl[:LABEL_25]
 call 177, stor[30], CURRENT_HP, stor[10]
 call 177, stor[30], MAX_HP, stor[11]
@@ -359,7 +359,7 @@
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp !&, 0x10, stor[8], inl[:LABEL_31]
 call 371, stor[0], stor[8], 0x0
-add 0x81, stor[8]
+add var[45][136], stor[8]
 or 0xf00, var[45][stor[8]]
 LABEL_31:
 return
@@ -1431,7 +1431,7 @@
 LABEL_54:
 mov var[32][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_55]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -1444,7 +1444,7 @@
 LABEL_57:
 mov var[32][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 gosub inl[:LABEL_209]
 incr stor[11]
 jmp_cmp <, stor[11], var[33], inl[:LABEL_57]
@@ -1704,11 +1704,8 @@
 call 238, 0x2, stor[0], 0x1, 0x0, 0x0, 0x0, 0x0
 rand 0x64, stor[8]
 jmp_cmp <, 0x64, stor[8], inl[:LABEL_77]
-call 497, 0xffffffff, stor[8]
-jmp_cmp ==, 0xffffffff, stor[8], inl[:LABEL_77]
-shl 0x18, stor[8]
-call 371, stor[0], stor[9], 0x0
-or stor[8], var[128 + stor[9]][7]
+call 497, null, reg[10]
+jmp_cmp ==, 0, stor[10], inl[:LABEL_77]
 mov 0x0, stor[8]
 mov 0x1e, stor[9]
 mov 0x0, stor[10]
@@ -2296,7 +2293,7 @@
 mov 0x0, stor[31]
 mov 0x0, stor[16]
 call 371, stor[32], stor[8], 0x0
-add 0xbd, stor[8]
+add var[45][196], stor[8]
 jmp_cmp &, 0x2, var[45][stor[8]], inl[:LABEL_132]
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp &, 0x10, stor[8], inl[:LABEL_132]
@@ -2924,7 +2921,7 @@
 call 169, stor[0], GUARD, stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_194]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 call 169, stor[0], STATUS, stor[8]
 and 0xff, stor[8]
 jmp_cmp ==, 0x0, stor[8], inl[:LABEL_197]
@@ -3025,7 +3022,7 @@
 gosub inl[:LABEL_200]
 jmp_cmp >, 0x0, stor[8], inl[:LABEL_208]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp &, 0x80, var[45][stor[9]], inl[:LABEL_206]
 or 0x80, var[45][stor[9]]
 call 600, stor[18]
@@ -3426,14 +3423,14 @@
 return
 LABEL_235:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_236]
 or 0x40, var[45][stor[9]]
 LABEL_236:
 return
 LABEL_237:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov 0x0, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
 and 0xffffff00, stor[8]
@@ -3588,18 +3585,14 @@
 mov var[128 + stor[25]][7], stor[26]
 shr 0x18, stor[26]
 and 0xff, stor[26]
-jmp_cmp ==, 0x0, stor[26], inl[:LABEL_246]
-call 498, stor[26], stor[8]
+jmp_cmp ==, 0x1, stor[26], inl[:RETURN_STOLEN_GOLD]
+jmp_cmp ==, 0x0, reg[10], inl[:LABEL_246]
+call 498, reg[10], reg[10]
 jmp inl[:LABEL_245]
-data 0x208
-data 0x5000005
-data 0x2000008
-data 0x218
-data 0x32
-data 0x2000008
-data 0x208
-data 0x2000008
-data 0x5000005
+RETURN_STOLEN_GOLD:
+mov var[5], stor[8]
+add 50, stor[8]
+mov stor[8], var[5]
 LABEL_245:
 mov 0x3, stor[8]
 mov 0x1e, stor[9]
