--- original
+++ modified
@@ -852,7 +852,7 @@
 div 0xa, stor[9]
 jmp_cmp >=, stor[9], stor[8], inl[:LABEL_18]
 call 371, stor[28], stor[10], 0x0
-add 0x81, stor[10]
+add var[45][136], stor[10]
 mov var[45][stor[10]], stor[9]
 jmp_cmp !&, 0x80, stor[9], inl[:LABEL_15]
 and 0x7, stor[9]
@@ -2574,7 +2574,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x208
 data 0x700092d
@@ -3288,7 +3288,7 @@
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_47]
 mov 0x0, stor[11]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov var[45][stor[9]], stor[10]
 jmp_cmp !&, 0x80, stor[10], inl[:LABEL_48]
 and 0xf, stor[10]
@@ -3311,7 +3311,7 @@
 call 600, stor[18]
 mov stor[0], stor[stor[stor[0], 18], 28]
 mov stor[32], stor[stor[stor[0], 18], 31]
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov stor[32], var[45][stor[9]]
 or 0x480, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
@@ -3356,7 +3356,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x208
 data 0x700092d
@@ -3396,7 +3396,7 @@
 data 0x200001f
 data 0x31f1200
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x208
 data 0x200001f
@@ -3446,7 +3446,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x208
 data 0x700092d
@@ -3476,7 +3476,7 @@
 data 0x200001c
 data 0x31c1200
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x208
 data 0x2000009
@@ -3745,7 +3745,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x211
 data 0x40
@@ -3753,7 +3753,7 @@
 data 0x49
 LABEL_67:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov 0x0, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
 and 0xfffffff0, stor[8]
@@ -3764,7 +3764,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x10c
 data 0x700092d
@@ -3788,7 +3788,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x210
 data 0xff7f
@@ -3804,7 +3804,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x210
 data 0xff7f
@@ -3821,7 +3821,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x49
 LABEL_68:
