--- original
+++ modified
@@ -24,7 +24,7 @@
 mov 0x20, var[128 + stor[8]][5]
 mov 0x0, var[128 + stor[8]][6]
 mov 0x0, var[128 + stor[8]][7]
-add 0x81, stor[8]
+add var[45][136], stor[8]
 mov 0x0, var[45][stor[8]]
 mov var[45][0], stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_2]
@@ -255,7 +255,7 @@
 LABEL_16:
 mov var[32][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_17]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -268,7 +268,7 @@
 LABEL_19:
 mov var[32][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 gosub inl[:LABEL_95]
 incr stor[11]
 jmp_cmp <, stor[11], var[33], inl[:LABEL_19]
@@ -953,7 +953,7 @@
 data 0x2000009
 data 0x1
 data 0x218
-data 0x45
+data 0xf00472d
 data 0x2000009
 data 0x218
 data 0x2000008
@@ -987,7 +987,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x10c
 data 0x700092d
@@ -1017,7 +1017,7 @@
 shr 0x1, stor[9]
 LABEL_67:
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 mov var[45][stor[10]], stor[10]
 jmp_cmp !&, 0x2, stor[10], inl[:LABEL_68]
 jmp_cmp &, 0x4, stor[10], inl[:LABEL_68]
@@ -1025,7 +1025,7 @@
 LABEL_68:
 jmp_cmp >=, 0x0, stor[9], inl[:LABEL_70]
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 jmp_cmp !&, 0x1, var[45][stor[10]], inl[:LABEL_70]
 call 177, stor[30], CURRENT_HP, stor[10]
 call 177, stor[30], MAX_HP, stor[11]
@@ -1575,7 +1575,7 @@
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp !&, 0x10, stor[8], inl[:LABEL_75]
 call 371, stor[0], stor[8], 0x0
-add 0x81, stor[8]
+add var[45][136], stor[8]
 or 0xf00, var[45][stor[8]]
 LABEL_75:
 return
@@ -1629,7 +1629,7 @@
 call 169, stor[0], GUARD, stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_80]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 call 169, stor[0], STATUS, stor[8]
 and 0xff, stor[8]
 jmp_cmp ==, 0x0, stor[8], inl[:LABEL_83]
@@ -1730,7 +1730,7 @@
 gosub inl[:LABEL_86]
 jmp_cmp >, 0x0, stor[8], inl[:LABEL_94]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp &, 0x80, var[45][stor[9]], inl[:LABEL_92]
 or 0x80, var[45][stor[9]]
 call 600, stor[18]
@@ -1767,7 +1767,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x148
 data 0x9000009
@@ -2131,14 +2131,14 @@
 return
 LABEL_121:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_122]
 or 0x40, var[45][stor[9]]
 LABEL_122:
 return
 LABEL_123:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov 0x0, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
 and 0xffffff00, stor[8]
@@ -2149,7 +2149,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -2174,7 +2174,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -2200,7 +2200,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x49
 LABEL_124:
