--- original
+++ modified
@@ -24,7 +24,7 @@
 mov 0x20, var[128 + stor[8]][5]
 mov 0x0, var[128 + stor[8]][6]
 mov 0x0, var[128 + stor[8]][7]
-add 0x81, stor[8]
+add var[45][136], stor[8]
 mov 0x0, var[45][stor[8]]
 mov var[45][0], stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_2]
@@ -278,7 +278,7 @@
 LABEL_11:
 mov var[32][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_12]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -291,7 +291,7 @@
 LABEL_14:
 mov var[32][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 gosub inl[:LABEL_46]
 incr stor[11]
 jmp_cmp <, stor[11], var[33], inl[:LABEL_14]
@@ -1188,7 +1188,7 @@
 data 0x2000009
 data 0x1
 data 0x218
-data 0x45
+data 0xf00472d
 data 0x2000009
 data 0x218
 data 0x2000008
@@ -1222,7 +1222,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x10c
 data 0x700092d
@@ -1276,7 +1276,7 @@
 data 0x200000a
 data 0x0
 data 0x218
-data 0xbd
+data 0xf00c42d
 data 0x200000a
 data 0x208
 data 0x7000a2d
@@ -1300,7 +1300,7 @@
 data 0x200000a
 data 0x0
 data 0x218
-data 0xbd
+data 0xf00c42d
 data 0x200000a
 data 0x70341
 data 0x1
@@ -1897,7 +1897,7 @@
 data 0x2000008
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000008
 data 0x211
 data 0xf00
@@ -1953,7 +1953,7 @@
 call 169, stor[0], GUARD, stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_31]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 call 169, stor[0], STATUS, stor[8]
 and 0xff, stor[8]
 jmp_cmp ==, 0x0, stor[8], inl[:LABEL_34]
@@ -2054,7 +2054,7 @@
 gosub inl[:LABEL_37]
 jmp_cmp >, 0x0, stor[8], inl[:LABEL_45]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp &, 0x80, var[45][stor[9]], inl[:LABEL_43]
 or 0x80, var[45][stor[9]]
 call 600, stor[18]
@@ -2091,7 +2091,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x148
 data 0x9000009
@@ -2455,7 +2455,7 @@
 return
 LABEL_72:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_73]
 or 0x40, var[45][stor[9]]
 LABEL_73:
@@ -2465,7 +2465,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x10c
 data 0x700092d
@@ -2489,7 +2489,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -2514,7 +2514,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -2540,7 +2540,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x49
 LABEL_74:
