--- original
+++ modified
@@ -24,7 +24,7 @@
 mov 0x20, var[128 + stor[8]][5]
 mov 0x0, var[128 + stor[8]][6]
 mov 0x0, var[128 + stor[8]][7]
-add 0x81, stor[8]
+add var[45][136], stor[8]
 mov 0x0, var[45][stor[8]]
 mov var[45][0], stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_2]
@@ -371,7 +371,7 @@
 LABEL_28:
 mov var[32][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_29]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -384,7 +384,7 @@
 LABEL_31:
 mov var[32][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 gosub inl[:LABEL_123]
 incr stor[11]
 jmp_cmp <, stor[11], var[33], inl[:LABEL_31]
@@ -599,7 +599,7 @@
 mov 0x0, stor[31]
 mov 0x0, stor[16]
 call 371, stor[32], stor[8], 0x0
-add 0xbd, stor[8]
+add var[45][196], stor[8]
 jmp_cmp &, 0x2, var[45][stor[8]], inl[:LABEL_52]
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp &, 0x10, stor[8], inl[:LABEL_52]
@@ -1150,7 +1150,7 @@
 data 0x2000009
 data 0x1
 data 0x218
-data 0x45
+data 0xf00472d
 data 0x2000009
 data 0x218
 data 0x2000008
@@ -1184,7 +1184,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x10c
 data 0x700092d
@@ -1214,7 +1214,7 @@
 shr 0x1, stor[9]
 LABEL_93:
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 mov var[45][stor[10]], stor[10]
 jmp_cmp !&, 0x2, stor[10], inl[:LABEL_94]
 jmp_cmp &, 0x4, stor[10], inl[:LABEL_94]
@@ -1222,7 +1222,7 @@
 LABEL_94:
 jmp_cmp >=, 0x0, stor[9], inl[:LABEL_96]
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 jmp_cmp !&, 0x1, var[45][stor[10]], inl[:LABEL_96]
 call 177, stor[30], CURRENT_HP, stor[10]
 call 177, stor[30], MAX_HP, stor[11]
@@ -1772,7 +1772,7 @@
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp !&, 0x10, stor[8], inl[:LABEL_101]
 call 371, stor[0], stor[8], 0x0
-add 0x81, stor[8]
+add var[45][136], stor[8]
 or 0xf00, var[45][stor[8]]
 LABEL_101:
 return
@@ -1820,7 +1820,7 @@
 call 169, stor[0], GUARD, stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_108]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 call 169, stor[0], STATUS, stor[8]
 and 0xff, stor[8]
 jmp_cmp ==, 0x0, stor[8], inl[:LABEL_111]
@@ -1921,7 +1921,7 @@
 gosub inl[:LABEL_114]
 jmp_cmp >, 0x0, stor[8], inl[:LABEL_122]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp &, 0x80, var[45][stor[9]], inl[:LABEL_120]
 or 0x80, var[45][stor[9]]
 call 600, stor[18]
@@ -1958,7 +1958,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x148
 data 0x9000009
@@ -2322,14 +2322,14 @@
 return
 LABEL_149:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_150]
 or 0x40, var[45][stor[9]]
 LABEL_150:
 return
 LABEL_151:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov 0x0, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
 and 0xffffff00, stor[8]
@@ -2340,7 +2340,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -2365,7 +2365,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -2391,7 +2391,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x49
 LABEL_152:
