--- original
+++ modified
@@ -24,7 +24,7 @@
 mov 0x20, var[128 + stor[8]][5]
 mov 0x0, var[128 + stor[8]][6]
 mov 0x0, var[128 + stor[8]][7]
-add 0x81, stor[8]
+add var[45][136], stor[8]
 mov 0x0, var[45][stor[8]]
 mov var[45][0], stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_2]
@@ -268,7 +268,7 @@
 LABEL_14:
 mov var[32][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_15]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -281,7 +281,7 @@
 LABEL_17:
 mov var[32][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 gosub inl[:LABEL_167]
 incr stor[11]
 jmp_cmp <, stor[11], var[33], inl[:LABEL_17]
@@ -496,7 +496,7 @@
 mov 0x0, stor[31]
 mov 0x0, stor[16]
 call 371, stor[32], stor[8], 0x0
-add 0xbd, stor[8]
+add var[45][196], stor[8]
 jmp_cmp &, 0x2, var[45][stor[8]], inl[:LABEL_39]
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp &, 0x10, stor[8], inl[:LABEL_39]
@@ -1947,7 +1947,7 @@
 data 0x2000009
 data 0x1
 data 0x218
-data 0x45
+data 0xf00472d
 data 0x2000009
 data 0x218
 data 0x2000008
@@ -1981,7 +1981,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x10c
 data 0x700092d
@@ -2011,7 +2011,7 @@
 shr 0x1, stor[9]
 LABEL_136:
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 mov var[45][stor[10]], stor[10]
 jmp_cmp !&, 0x2, stor[10], inl[:LABEL_137]
 jmp_cmp &, 0x4, stor[10], inl[:LABEL_137]
@@ -2019,7 +2019,7 @@
 LABEL_137:
 jmp_cmp >=, 0x0, stor[9], inl[:LABEL_139]
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 jmp_cmp !&, 0x1, var[45][stor[10]], inl[:LABEL_139]
 call 177, stor[30], CURRENT_HP, stor[10]
 call 177, stor[30], MAX_HP, stor[11]
@@ -2569,7 +2569,7 @@
 call 169, stor[0], STATUS, stor[8]
 jmp_cmp !&, 0x10, stor[8], inl[:LABEL_144]
 call 371, stor[0], stor[8], 0x0
-add 0x81, stor[8]
+add var[45][136], stor[8]
 or 0xf00, var[45][stor[8]]
 LABEL_144:
 return
@@ -2613,7 +2613,7 @@
 call 169, stor[0], GUARD, stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_152]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 call 169, stor[0], STATUS, stor[8]
 and 0xff, stor[8]
 jmp_cmp ==, 0x0, stor[8], inl[:LABEL_155]
@@ -2714,7 +2714,7 @@
 gosub inl[:LABEL_158]
 jmp_cmp >, 0x0, stor[8], inl[:LABEL_166]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp &, 0x80, var[45][stor[9]], inl[:LABEL_164]
 or 0x80, var[45][stor[9]]
 call 600, stor[18]
@@ -2751,7 +2751,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x148
 data 0x9000009
@@ -3115,7 +3115,7 @@
 return
 LABEL_193:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_194]
 or 0x40, var[45][stor[9]]
 LABEL_194:
@@ -3125,7 +3125,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x10c
 data 0x700092d
@@ -3149,7 +3149,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -3174,7 +3174,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -3200,7 +3200,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x49
 LABEL_195:
