--- original
+++ modified
@@ -24,7 +24,7 @@
 mov 0x20, var[128 + stor[8]][5]
 mov 0x0, var[128 + stor[8]][6]
 mov 0x0, var[128 + stor[8]][7]
-add 0x81, stor[8]
+add var[45][136], stor[8]
 mov 0x0, var[45][stor[8]]
 mov var[45][0], stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_2]
@@ -554,7 +554,7 @@
 LABEL_17:
 mov var[48][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_18]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -567,7 +567,7 @@
 LABEL_20:
 mov var[48][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov var[45][stor[9]], stor[10]
 and 0xf, stor[10]
 jmp_cmp ==, 0x0, stor[10], inl[:LABEL_21]
@@ -2020,7 +2020,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x10c
 data 0x700092d
@@ -2044,7 +2044,7 @@
 call 168, stor[30], var[45][64], ADDITION_HITS
 call 481, stor[30], stor[28], stor[9], stor[10]
 call 371, stor[30], stor[11], 0x0
-add 0x81, stor[11]
+add var[45][136], stor[11]
 mov var[45][stor[11]], stor[12]
 and 0xf, stor[12]
 jmp_cmp !=, 0x3, stor[12], inl[:LABEL_105]
@@ -2053,7 +2053,7 @@
 incr stor[9]
 LABEL_105:
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 mov var[45][stor[10]], stor[10]
 and 0x2, stor[10]
 jmp_cmp ==, 0x0, stor[10], inl[:LABEL_106]
@@ -2063,7 +2063,7 @@
 LABEL_106:
 jmp_cmp >=, 0x0, stor[9], inl[:LABEL_108]
 call 371, stor[30], stor[10], 0x0
-add 0xbd, stor[10]
+add var[45][196], stor[10]
 jmp_cmp !&, 0x1, var[45][stor[10]], inl[:LABEL_108]
 call 169, stor[30], CURRENT_HP, stor[10]
 call 169, stor[30], MAX_HP, stor[11]
@@ -2567,7 +2567,7 @@
 call 169, stor[0], GUARD, stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_110]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 call 169, stor[0], STATUS, stor[8]
 and 0xff, stor[8]
 jmp_cmp ==, 0x0, stor[8], inl[:LABEL_111]
@@ -2629,7 +2629,7 @@
 gosub inl[:LABEL_113]
 jmp_cmp >, 0x0, stor[8], inl[:LABEL_119]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp &, 0x80, var[45][stor[9]], inl[:LABEL_118]
 or 0x80, var[45][stor[9]]
 call 600, stor[18]
@@ -2654,7 +2654,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x148
 data 0x9000009
@@ -2956,14 +2956,14 @@
 return
 LABEL_141:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_142]
 or 0x40, var[45][stor[9]]
 LABEL_142:
 return
 LABEL_143:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov 0x0, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
 and 0xffffff00, stor[8]
@@ -2974,7 +2974,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -2999,7 +2999,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -3025,7 +3025,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x49
 LABEL_144:
