--- original
+++ modified
@@ -24,7 +24,7 @@
 mov 0x20, var[128 + stor[8]][5]
 mov 0x0, var[128 + stor[8]][6]
 mov 0x0, var[128 + stor[8]][7]
-add 0x81, stor[8]
+add var[45][136], stor[8]
 mov 0x0, var[45][stor[8]]
 mov var[45][0], stor[8]
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_2]
@@ -759,7 +759,7 @@
 LABEL_43:
 mov var[48][stor[11]], stor[10]
 call 371, stor[10], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 jmp_cmp !&, 0x80, var[45][stor[9]], inl[:LABEL_44]
 and 0xff7f, var[45][stor[9]]
 or 0x10000, var[45][stor[9]]
@@ -772,7 +772,7 @@
 LABEL_46:
 mov var[48][stor[11]], stor[28]
 call 371, stor[28], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov var[45][stor[9]], stor[10]
 and 0xf, stor[10]
 jmp_cmp ==, 0x0, stor[10], inl[:LABEL_47]
@@ -1902,7 +1902,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x10c
 data 0x700092d
@@ -1926,7 +1926,7 @@
 call 168, stor[30], var[45][64], ADDITION_HITS
 call 481, stor[30], stor[28], stor[9], stor[10]
 call 371, stor[30], stor[11], 0x0
-add 0x81, stor[11]
+add var[45][136], stor[11]
 mov var[45][stor[11]], stor[12]
 and 0xf, stor[12]
 jmp_cmp !=, 0x3, stor[12], inl[:LABEL_85]
@@ -1981,7 +1981,7 @@
 jmp_cmp !=, 0x0, stor[8], inl[:LABEL_87]
 mov 0x0, stor[11]
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov var[45][stor[9]], stor[10]
 jmp_cmp !&, 0x80, stor[10], inl[:LABEL_88]
 and 0xf, stor[10]
@@ -2005,7 +2005,7 @@
 call 600, stor[18]
 mov stor[0], stor[stor[stor[0], 18], 28]
 mov stor[32], stor[stor[stor[0], 18], 31]
-add 0x81, stor[9]
+add var[45][136], stor[9]
 and 0xfffffff0, var[45][stor[9]]
 or stor[32], var[45][stor[9]]
 or 0x380, var[45][stor[9]]
@@ -2051,7 +2051,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x208
 data 0x700092d
@@ -2095,7 +2095,7 @@
 data 0x200001f
 data 0x31f1200
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x210
 data 0x1000000
@@ -2147,7 +2147,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x208
 data 0x700092d
@@ -2172,7 +2172,7 @@
 call 371, stor[28], stor[9], 0x0
 call 600, stor[18]
 mov stor[28], stor[stor[stor[0], 18], 28]
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov stor[9], stor[stor[stor[0], 18], 9]
 or 0x80, var[45][stor[9]]
 and 0xfffeffff, var[45][stor[9]]
@@ -2465,12 +2465,12 @@
 data 0x49
 LABEL_112:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 or 0x40, var[45][stor[9]]
 return
 LABEL_113:
 call 371, stor[0], stor[9], 0x0
-add 0x81, stor[9]
+add var[45][136], stor[9]
 mov 0x0, var[45][stor[9]]
 call 169, stor[0], STATUS, stor[8]
 and 0xffffff00, stor[8]
@@ -2481,7 +2481,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -2506,7 +2506,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x70341
 data 0x1000000
@@ -2532,7 +2532,7 @@
 data 0x2000009
 data 0x0
 data 0x218
-data 0x81
+data 0xf00882d
 data 0x2000009
 data 0x49
 LABEL_114:
